--- rt-stubs/src/json/tape.rs
+++ rt-stubs/src/json/tape.rs
@@ -235,6 +235,8 @@
     if tag != 0xFFF6_0000_0000_0000 && tag != 0xFFFC_0000_0000_0000 && tag != 0xFFFE_0000_0000_0000 {
         return;
     }
+    
+    libc::printf("DEBUG: __bs_prop_set called with tag: %lx\n\0".as_ptr() as *const libc::c_char, tag);
     
     let payload = obj_tagged & 0x0000_FFFF_FFFF_FFFF;
     if payload == 0 {
